### PR TITLE
Rename number of issues

### PR DESCRIPTION
**Pull request by KyleMacPherson**
Rename `number_of_issues` to `feedex_comments` for clarity.

### CODE REVIEW

File: db/migrate/20180309144814_rename_number_of_issues_to_feedex_comments.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class RenameNumberOfIssuesToFeedexComments < ActiveRecord::Migration[5.1]
+  def change
+    rename_column :facts_metrics, :feedex_comments, :feedex_comments
```
Comment (pmanrubia) [inline db/migrate/20180309144814_rename_number_of_issues_to_feedex_comments.rb]: Not sure the rename is valid?

**Comment by KyleMacPherson** [comment]
[(1) Rename `number_of_issues` to `feedex_comments` (Part of EPIC #216)](https://trello.com/c/txIU2saj/180-rename-numberofissues-to-feedexcomments)

File: db/migrate/20180309144814_rename_number_of_issues_to_feedex_comments.rb (hunk 1 above)
Comment (KyleMacPherson) [reply db/migrate/20180309144814_rename_number_of_issues_to_feedex_comments.rb]: Sorry @pmanrubia, well spotted.  I should not have used `find_and_replace`! 😂 😔

File: spec/requests/api/v1/metrics_spec.rb [hunk 2]
```diff
@@ -92,20 +92,20 @@
     end
 
     it 'returns `feedex issues` between two dates' do
-      get "/api/v1/metrics/number_of_issues/#{content_id}/time-series", params: { from: '2018-01-13', to: '2018-01-15' }
+      get "/api/v1/metrics/feedex_comments/#{content_id}/time-series", params: { from: '2018-01-13', to: '2018-01-15' }
```
Comment (pmanrubia) [inline spec/requests/api/v1/metrics_spec.rb]: In this case we are changing an API endpoint that we shouldn't change (we are changing the URL of an API of another system)

Comment (pmanrubia) [reply spec/requests/api/v1/metrics_spec.rb]: Sorry, @KyleMacPherson my bad. This is ok, as it is our internal API.
Discard this message, please.

**Pull request merged**